widget/embed_ox_talks.js

```diff
@@ -194,5 +303,39 @@ var oxtalks = {
             selector = this.createContainer();
         }
         this.queryTalks(params, this.buildList, $(selector));
-    }
+    },
+
+    //Create a calendar using fullCalendar. The query will be invoked by fullCalendar
+    showCalendar: function(params, selector) {
+        //append to body if no selector specified
+        if (!selector) {
+            selector = this.createContainer();
+        }
+        
+        //check that the fullcalendar plugin is present and provide helpful feedback if not
+        if(!$.fullCalendar)
+        {
+            console.error("Fullcalendar plugin not present. Please consult the docs at http://talksox.readthedocs.org/en/latest/");
+            //error message for the end user
+            $(selector).text("Unable to display calendar");
+            return;
+        }
+
+        //Create calendar
+        var eventSource = this.createEventSource(params);
+        $(selector).fullCalendar({
+            eventSources: [ eventSource ],
+            eventColor: '#002147',
```

Comment:
Not sure how atm but people might want to change colors / style at some points, maybe add an issue to see how this could be improved... (Possibly have a default set of options but allow overriding)